index.html

```diff
@@ -355,6 +355,12 @@ <h1>Contact Me</h1>
         </div>
     </footer>
```

Comment:
- [ Optional ] Kindly adjust the empty space below the contact section just the like the figma design .

Your Footer look likes this in mobile and desktop screen size.
![Capture](https://user-images.githubusercontent.com/93598189/195142177-9498d15f-21b5-4b44-9874-b2dec156f403.PNG)